useCases/GetCommitmentsAndExportFile/ConcatTwoArrays/index.ts

```diff
@@ -0,0 +1,3 @@
+export const concatArrays = (array1, array2) => {
+  return array1.concat(array2);
```

Comment:
What do you think of validating if the params aren't null?

Reply:
Done.